use/accounts.js

```diff
@@ -10,85 +12,88 @@ export const useAccounts = () => {
   const accountFields = [
     {
       name: 'name',
-      key: 'account.name',
+      label: t('account.name'),
       required: true,
       event: true,
     },
     {
       name: 'email',
-      key: 'account.email',
+      label: t('account.email'),
       register: true,
       required: true,
       event: true,
     },
     {
       name: 'password',
       type: 'password',
-      label: 'New Password',
+      label: t('account.password'),
       register: true,
     },
     {
       name: 'phone',
-      key: 'account.phone',
+      label: t('account.phone'),
       required: true,
       event: true,
     },
     {
       name: 'zone',
-      label: 'Timezone',
+      label: t('account.zone'),
       type: 'richselect',
       options: moment.tz.names(),
       require: true,
     },
     {
       name: 'newsletter',
-      key: 'account.newsletter',
+      label: t('account.newsletter.label'),
       type: 'multi',
       options: [
         {
           value: 'party',
-          label: 'Social/party events',
+          label: t('account.newsletter.options.party'),
         },
         {
           value: 'class',
-          label: 'Dance classes',
+          label: t('account.newsletter.options.class'),
         },
         {
           value: 'workshop',
-          label: 'Workshops and bootcamps',
+          label: t('account.newsletter.options.workshop'),
         },
         {
           value: 'online',
-          label: 'Online dance classes',
+          label: t('account.newsletter.options.online'),
         },
         {
           value: 'festival',
-          label: 'International festivals',
+          label: t('account.newsletter.options.festival'),
         },
       ],
     },
     {
       name: 'comment',
-      label: 'Comment',
+      label: t('account.comment'),
       type: 'text',
       event: true,
     },
     {
       name: 'withPartner',
       type: 'select',
-      label: 'Do you have a partner?',
-      options: ['Yes', 'No'],
+      label: t('account.withPartner.label'),
+      options: [
```

Comment:
import and reuse this object instead

```suggestion
      options: booleanOptions
```